flink-libraries/flink-table/src/test/scala/org/apache/flink/table/utils/CommonTestData.scala

```diff
@@ -68,6 +62,90 @@ object CommonTestData {
     )
   }
 
+  def getMockedFlinkExternalCatalog: ExternalCatalog = {
+    val csvRecord1 = Seq(
```

Comment:
val csvRecord1 = Seq() is just csv data, maybe it's better to name it as csvDataRecords?